pkg/tracing/tracing.go

```diff
@@ -144,14 +151,34 @@ func (t *tracerProvider) reinitialize() {
 
 func createTracerProvider(service string, cfg *config.Tracing, user, pass string) (trace.TracerProvider, error) {
 	if !cfg.Enabled {
-		return trace.NewNoopTracerProvider(), nil
+		return noop.NewTracerProvider(), nil
+	}
+	u, err := url.Parse(cfg.Endpoint)
```

Comment:
Jaeger exporter expected a complete URL path like `http://website.com:port/path`, while otlp exporter accepts  host and path as separate arguments. This URL parsing is added to ensure that user experience has not changed.